lib/index.ts

```diff
@@ -455,8 +455,8 @@ export async function buildDepTreeFromImportsAndModules(root: string = '.') {
   // TODO(BST-657): parse go.mod file to obtain root module name and go version
 
   const depTree: DepTree = {
-    name: '.', // The correct name should come from the go.mod parser
-    version: 'unknown', // TODO(BST-657): try `git describe`?
+    name: path.basename(root), // The correct name should come from the `go list` command
+    version: '0.0.0', // TODO(BST-657): try `git describe`?
```

Comment:
is that the standard we have with all the other package managers?

Reply:
Yep, that's the reason.